app/controllers/api/api_controller.rb

```diff
@@ -1,2 +1,18 @@
 class Api::ApiController < ActionController::API
-end
\ No newline at end of file
+  include JsonWebToken
+
+  before_action :authenticate_request
+
+  private
+
+  def authenticate_request
+    header = request.headers['Authorization']
+    header = header.split(' ').last if header
+    @decoded = jwt_decode(header)
+    begin
+      @current_user = User.find(@decoded[:user_id])
+    rescue ActiveRecord::RecordNotFound
+      render json: {error: "Unauthorized. Plase, make sure you are logged."}, status: :unauthorized
```

Comment:
Grammar is a bit off here. It should be:
```suggestion
      render json: {error: "Unauthorized. Please, make sure you are logged in."}, status: :unauthorized
```

Although, to me, "Unauthorized" is enough